lib/netsuite.rb

```diff
@@ -9,7 +9,13 @@
 module NetSuite
   autoload :Configuration,    'netsuite/configuration'
   autoload :Response,         'netsuite/response'
-  autoload :AsyncStatus,      'netsuite/async_status'
+  autoload :Status,           'netsuite/status'
+
+  module Async
+    autoload :AsyncStatus,        'netsuite/async/async_status'
```

Comment:
I removed the 'Async' class name prefix.  Also renamed WriteResults to
WriteResponseList, and WriteResult to WriteResponse, to match the WSDL.

On Fri, Feb 27, 2015 at 8:00 PM, Patrick Burke <patrick.burke@shopify.com>
wrote:

> Actually maybe not...these class names match those in the schema.  I agree
> they don't look nice.  I'll sleep on it.
> On Feb 27, 2015 7:58 PM, "Patrick Burke" <patrick.burke@shopify.com>
> wrote:
>
>> Yep.  :)
>> On Feb 27, 2015 7:45 PM, "Jerome Cornet" <notifications@github.com>
>> wrote:
>>
>>> In lib/netsuite.rb
>>> <https://github.com/prburke/netsuite/pull/3#discussion_r25550607>:
>>>
>>> > @@ -9,7 +9,13 @@
>>> >  module NetSuite
>>> >    autoload :Configuration,    'netsuite/configuration'
>>> >    autoload :Response,         'netsuite/response'
>>> > -  autoload :AsyncStatus,      'netsuite/async_status'
>>> > +  autoload :Status,           'netsuite/status'
>>> > +
>>> > +  module Async
>>> > +    autoload :AsyncStatus,        'netsuite/async/async_status'
>>>
>>> should this be Async::Status instead of Async::AsyncStatus ?
>>>
>>> —
>>> Reply to this email directly or view it on GitHub
>>> <https://github.com/prburke/netsuite/pull/3/files#r25550607>.
>>>
>>